### ISSUE TITLE
Command ```newrelic reporting junit``` does not work anymore

### ISSUE THREAD

**Issue by dhoppe** [newrelic/junit-reporter-action#3]
## Description

This GitHub Action uses a deprecated environment variable ```NEW_RELIC_INSIGHTS_INSERT_KEY```.

## Steps to Reproduce

```
---
name: Release

on:
  - release

jobs:
  newrelic:
    runs-on: ubuntu-latest
    name: Basic Usage
    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Post JUnit test results to New Relic
        uses: newrelic/junit-reporter-action@v0.1
        with:
          accountId: ${{ secrets.NEW_RELIC_ACCOUNT_ID }}
          insertApiKey: ${{ secrets.NEW_RELIC_INSERT_API_KEY }}
          testOutputPath: test-output/integration.xml
```

## Expected Behavior

The GitHub Action should run without any error messages. 

## Relevant Logs / Console output

```
level=fatal msg="a License key is required, set one in your default profile or use the NEW_RELIC_LICENSE_KEY environment variable"
```

## Additional context

This GitHub Action has not been updated for almost 2 years, but the Dockerfile is still referencing to the latest version of the Newrelic CLI. 
https://github.com/newrelic/junit-reporter-action/blob/main/Dockerfile#L1

The `entrypoint.sh` does not provide a real value for `NEW_RELIC_API_KEY` and `NEW_RELIC_INSIGHTS_INSERT_KEY` is not being used anymore. Instead `NEW_RELIC_LICENSE_KEY` should be used. 
https://github.com/newrelic/junit-reporter-action/blob/main/entrypoint.sh#L3-L4

**Comment by jsantinipax8** [comment]
@kidk Can this version be re-opened? It doesn't look like version `0.2.0` is published to the marketplace unless I'm missing something?  Right now it still shows `0.1.1`

**Comment by kidk** [comment]
@jsantinipax8 Fixed, the latest version is now 0.2.0

**Comment by jsantinipax8** [comment]
Thank You!
